### PR TITLE
Code review

### PR DESCRIPTION
**Pull request by tshemsedinov**

### CODE REVIEW

File: client/client.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const http = require('./common');
```
Comment (tshemsedinov) [inline client/client.js:3]: That's strange that common exports http

File: client/client.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+const Client = (baseUrl) => {
```
Comment (tshemsedinov) [inline client/client.js:5]: We name functions in lowerCamelCase

File: server/application/Match/finishMatch.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+module.exports = async (
+  id,
+  { homeGoals, guestGoals },
+  { TeamRepository, MatchRepository },
+) => {
+  const match = await MatchRepository.readById(id);
+  const homeTeam = await TeamRepository.readById(match.homeId);
+  const guestTeam = await TeamRepository.readById(match.guestId);
```
Comment (tshemsedinov) [inline server/application/Match/finishMatch.js:10]: You can do this in parallel

File: server/application/Match/index.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+const finishMatch = require('./finishMatch');
+
+module.exports = {
+  getMatches,
+  getMatchById,
+  createMatch,
+  deleteMatch,
+  updateMatch,
+  finishMatch,
+};
```
Comment (tshemsedinov) [inline server/application/Match/index.js:17]: Looks like a boilerplate for all entities

**Comment by SlavaIvanchenko17** [comment]
)

**Pull request closed without merging**